libkbfs/crypto_common.go

```diff
@@ -190,10 +196,24 @@ func (c CryptoCommon) EncryptBlock(block Block, key kbfscrypto.BlockCryptKey) (
 		return -1, kbfscrypto.EncryptedBlock{}, err
 	}
 
-	encryptedBlock, err =
-		kbfscrypto.EncryptPaddedEncodedBlock(paddedBlock, key)
-	if err != nil {
-		return -1, kbfscrypto.EncryptedBlock{}, err
+	switch c.blockCryptVersioner.BlockCryptVersion() {
```

Comment:
ah, I see, this is where the switch happens. Yeah, might be cleaner to move this logic into `kbfscrypto.EncryptPaddedEncodedBlock(..., version ...)`, but I don't feel too strongly about it

Reply:
Will fix in a future cleanup PR.